### PR TITLE
doc: native_posix: Add mention about virtual USB

### LINKED ISSUE #13232: native_posix doc: Add mention of virtual USB
**Is your enhancement proposal related to a problem? Please describe.**
In #9877 a USB driver for native_posix was added, together with a small description in the USB subsystem documentation. This is very nice.
But no mention of this driver was added to the native_posix board documentation itself.

**Describe the solution you'd like**
It would be nice having a small paragraph (similar to the one for ethernet*1) in the peripherals section of native_posix, which would link to the already existing section in the USB subsys doc. 

*1 : https://docs.zephyrproject.org/latest/boards/posix/native_posix/doc/board.html#peripherals

**Describe alternatives you've considered**
None.

### PR DESCRIPTION
**Pull request by finikorg**
Add information and link to Virtual USB controller.

Fixes #13232.

Signed-off-by: Andrei Emeltchenko <andrei.emeltchenko@intel.com>

### CODE REVIEW

File: boards/posix/native_posix/doc/index.rst [hunk 1]
```diff
@@ -516,6 +516,10 @@ The following peripherals are currently provided with this board:
   must be powered down and support Bluetooth Low Energy (i.e. support the
   Bluetooth specification version 4.0 or greater).
 
+**USB controller**:
+  It's possible to use Virtual USB controller working over USB/IP protocol.
```
Comment (aescolar) [inline boards/posix/native_posix/doc/index.rst]: Would an article would be missing here : "..to use +the+ Virtual USB.."

**Review by aescolar — CHANGES_REQUESTED** [review]
Thanks @finikorg. Just a minor nit from me

File: boards/posix/native_posix/doc/index.rst (hunk 1 above)
Comment (finikorg) [reply boards/posix/native_posix/doc/index.rst]: Done

**Review by aescolar — APPROVED** [review]

File: boards/posix/native_posix/doc/index.rst [hunk 2]
```diff
@@ -516,6 +516,10 @@ The following peripherals are currently provided with this board:
   must be powered down and support Bluetooth Low Energy (i.e. support the
   Bluetooth specification version 4.0 or greater).
 
+**USB controller**:
+  It's possible to use the Virtual USB controller working over USB/IP protocol.
+  More information can be found in :ref:`Testing USB over USP/IP in native_posix`.
```
Comment (aescolar) [inline boards/posix/native_posix/doc/index.rst]: This one ```:ref:`Testing USB over USP/IP in native_posix` ``` is not something you can refer from other docs.
You need to add a reference on it, something like:
(that line is over 80 chars by the way)
```diff
diff --git a/boards/posix/native_posix/doc/index.rst b/boards/posix/native_posix/doc/index.rst
index 2c4c8c1e89..8712b1a0a4 100644
--- a/boards/posix/native_posix/doc/index.rst
+++ b/boards/posix/native_posix/doc/index.rst
@@ -517,8 +517,9 @@ The following peripherals are currently provided with this board:
   Bluetooth specification version 4.0 or greater).
 
 **USB controller**:
-  It's possible to use the Virtual USB controller working over USB/IP protocol.
-  More information can be found in :ref:`Testing USB over USP/IP in native_posix`.
+  It's possible to use the Virtual USB controller working over the USB/IP
+  protocol. More information can be found in
+  :ref:`Testing USB over USP/IP in native_posix <testing_USB_native_posix>`.
 
 **Display driver**:
   A display driver is provided that creates a window on the host machine to
diff --git a/doc/reference/usb/index.rst b/doc/reference/usb/index.rst
index a1a794834d..8760ef468f 100644
--- a/doc/reference/usb/index.rst
+++ b/doc/reference/usb/index.rst
@@ -123,6 +123,8 @@ the vendor requests:
 The class driver waits for the :makevar:`USB_DC_CONFIGURED` device status code
 before transmitting any data.
 
+.. _testing_USB_native_posix:
+
 Testing USB over USP/IP in native_posix
 ***************************************
```

**Review by aescolar — CHANGES_REQUESTED** [review]
I was too quick accepting, sorry, please see below

File: boards/posix/native_posix/doc/index.rst (hunk 2 above)
Comment (finikorg) [reply boards/posix/native_posix/doc/index.rst]: Sorry for that, somehow I could not build docs locally, I thought CI will show errors if any.

Comment (finikorg) [reply boards/posix/native_posix/doc/index.rst]: I believe I fixed this now.

**Review by aescolar — APPROVED** [review]

**Pull request merged**